setup.cfg

```diff
@@ -40,6 +40,8 @@ install_requires =
     retro-data-structures>=0.6.0
     SuperDuperMetroid>=1.2.0
     open-dread-rando>=0.2.0
+    cave-story-randomizer>=0.8.7
+    tsc-utils>=0.1.2
```

Comment:
Where did you use this one?

Reply:
https://github.com/randovania/randovania/pull/2420/files#diff-1693ee3ac812314d57b7f1c569701a6e5e56b340bf2b94107f4314272fdc7960R145

It'll be used later for starting items, the connector, etc. as well. Converting a number to a legal TSC value, especially a negative number, is shockingly complicated lol so it's worthy of its own package.

Reply:
It's now being used to initialize the HP counter in-game according to the configuration's starting HP.